platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/Capture.kt

```diff
@@ -96,22 +147,28 @@ object Capture {
                 return
             }
 
-            // If the logger has already been configured, do nothing.
-            if (default.get() != null) {
-                Log.w("capture", "Attempted to initialize Capture more than once")
-                return
+            // Ideally we would use `getAndUpdate` in here but it's available for API 24 and up only.
+            when (default.getAndSet(LoggerState.ConfigurationStarted)) {
+                is LoggerState.NotConfigured -> {
+                    try {
+                        val logger = LoggerImpl(
+                            apiKey = apiKey,
+                            apiUrl = apiUrl,
+                            fieldProviders = fieldProviders,
+                            dateProvider = dateProvider ?: SystemDateProvider(),
+                            configuration = configuration,
+                            sessionStrategy = sessionStrategy,
+                            bridge = bridge,
+                        )
+                        default.set(LoggerState.Configured(logger))
+                    } catch (e: Throwable) {
+                        Log.w("capture", "Capture initialization failed")
```

Comment:
```suggestion
                        Log.w("capture", "Capture initialization failed", e)
```

Reply:
^ we should attach the root cause so at least they have some info about what happened

Reply:
this is to distinguish other possible root causes that we're not anticipating (see my other comment)